### PR TITLE
Add graphs for demographical data

### PR DESCRIPTION
**Pull request by ihsan314**
Add graphs of these properties to Univiz:
- Admission rate
- Average SAT score
- Number of undergraduate students enrolled
- The ratio of male and female students

Here's what they look like:
image

### CODE REVIEW

**Comment by ihsan314** [comment]
> Can you add a screenshot of the graphs on the `dashboard.html` to the PR description so that we know it shows up?

I've now edited the PR description to show the graphs.

File: capstone/src/main/webapp/js/demographicalGraphs.js [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+async function deserializeNumUndergradsData() {
+  const data = new google.visualization.DataTable();
+  await fetch('/viz/stats')
+      .then((response) => response.json())
+      .then(
+          (numUndergradsInfo) =>
+            populateNumUndergradsDataTable(data, numUndergradsInfo),
+      );
+  return data;
+}
```
Comment (ccondit) [inline capstone/src/main/webapp/js/demographicalGraphs.js:96]: This is going to result in a REST call for every graph. Is there a way we can consolidate those network calls to save resources?

I'm fine to address this in a follow up...

Comment (ihsan314) [reply capstone/src/main/webapp/js/demographicalGraphs.js:96]: Ah yea I was planning on addressing this later. I think what I'll try is fetching and passing a parameter consisting of the JSON object. Then not only will network calls be reduced, but all the populate functions can run in parallel.

Comment (kaveh096) [reply capstone/src/main/webapp/js/demographicalGraphs.js:96]: +1 to reducing network round trips.

**Pull request merged**